xtask/public-api/aya-ebpf.txt

```diff
@@ -2833,6 +2833,3 @@ pub fn aya_ebpf::programs::tracepoint::TracePointContext::as_ptr(&self) -> *mut
 impl aya_ebpf::EbpfContext for aya_ebpf::programs::xdp::XdpContext
 pub fn aya_ebpf::programs::xdp::XdpContext::as_ptr(&self) -> *mut core::ffi::c_void
 pub fn aya_ebpf::check_bounds_signed(value: i64, lower: i64, upper: i64) -> bool
-#[no_mangle] pub unsafe c fn aya_ebpf::memcpy(dest: *mut u8, src: *mut u8, n: usize)
```

Comment:
this means we're generating the API for the wrong arch right?

Reply:
Absolutely, but we always have. We are generating the API for x86_64-unknown-linux-gnu.

Perhaps we can do better in a follow up, but that's where we're at presently.